### DISCUSSION TITLE
Row grouping

### DISCUSSION THREAD

**Discussion by krishnamoorthy51** [l-lin/angular-datatables#1675] [Q&A]
hii @shanmukhateja @Jared314 is that row grouping is available for angular 2+ 
if available means can u provide the example for that

**Comment by shanmukhateja** [comment]
Hi,

You can achieve this using https://datatables.net/examples/advanced_init/row_grouping.html

Note: Use https://l-lin.github.io/angular-datatables/#/basic/with-options as base and adapt it to row_grouping example from datatables.net

**Reply by krishnamoorthy51** [reply]
@shanmukhateja i have done this thanks for your support 
i have another query is that possible to hide the rows when we click the group heading ???

**Reply by shanmukhateja** [reply]
Sorry, I don't know how to do that.
